docs/src/terminology.md

```diff
@@ -16,6 +20,10 @@ A front-end application that interacts with a Solana cluster.
 
 The result of interpreting all programs on the ledger at a given [tick height](terminology.md#tick-height). It includes at least the set of all [accounts](terminology.md#account) holding nonzero [native tokens](terminology.md#native-tokens).
 
+## batches
```

Comment:
I'm not sure we need this definition. A batch is just a set of something. Looks like you're defining a batch of shreds here, but there's also transaction batches.  Where are you seeing this term exactly?

Reply:
Was synthesized from here: https://docs.solana.com/cluster/overview#confirming-transactions

Reply:
Ah, how about changing that doc to use the term "shreds", which is the term used in the codebase?